### PR TITLE
new test case to secret/strategy.go

### PR DESCRIPTION
**Pull request by tianshapjq**
new test case to pkg/registry/core/secret/strategy.go

### CODE REVIEW

**Comment by tianshapjq** [comment]
@justinsb @juanvallejo hi, can this pr be merged?

File: pkg/registry/core/secret/strategy_test.go
Comment (soltysh) [inline pkg/registry/core/secret/strategy_test.go]: Move up, to the group where all golang imports are. The below imports should go up to k8s imports as well.

**Review by soltysh — APPROVED** [review]
One stylistic nit, otherwise lgtm. But you need approval from one of the registry approvers.

**Review by juanvallejo — APPROVED** [review]

**Pull request closed without merging**
